### DISCUSSION TITLE
Navigation items

### DISCUSSION THREAD

**Discussion by eaziweb95** [filamentphp/filament#2902] [Help] [answered]
Hi
I'd like to access from the same Model (Demande) to 2 differents forms. 
One Wizard form for creation and another FieldSetForm for view and update.
I duplicated the DemandeResource files and rename with DemandeResourceFront

![DemandeFront](https://user-images.githubusercontent.com/58070424/175284498-9977624f-a9b5-4ed6-bc72-20cdb6f13441.jpg)

And put the DemandeFrontResource on another Navigation Group, it appears like that

![navigation](https://user-images.githubusercontent.com/58070424/175284831-2e12806e-328a-4907-9fb2-39a5fb1b05b9.jpg)

The result is the same when i click on Demande or Demande Usager, they use the DemandeResrouce

how can i switch from Demande Usager to DemandeFrontResource ?

**Comment by eaziweb95** [comment]
Thanks for your reply

**Comment by danharrin** [comment]
Each resource needs a unique `$slug` property

**Comment by eaziweb95** [comment, marked as answer]
ok but how can i do to access to 2 differents forms on the same Model ?

**Reply by studiowizjo** [reply]
If you got two separate resources, then in each one you can create different form. 

You can also have only one resource, remove from  `DemandeResource.php` method `form()` and create it in `CreateDemande.php` and `EditDemande.php` with separate fields and logic.

**Reply by eaziweb95** [reply]
ok i will try this solution, thanks a lot !

**Reply by eaziweb95** [reply]
It works thanks !
